README.md

```diff
@@ -16,6 +16,11 @@ includes Shields-related and non-Shields-related resources._
 - [NodeICO](https://nodei.co/) &ndash; Large-format status badges for Node.js
   projects &ndash; which were once very popular!
 
+### Badge tools
+
+- [Badgie](https://badgie.me/) &ndash; Adds badges to your project based on
+  what it is.
```

Comment:
"based on what it is" doesn't really explain this to me well as a new user. How about "based on scanning the repository content" or something like that?

Reply:
Hi @chris48s, thanks for the feedback! I just updated the description. What do you think?